packages/react/src/QuestionnaireForm/QuestionnaireForm.tsx

```diff
@@ -174,13 +175,14 @@ function QuestionnaireFormItemArray(props: QuestionnaireFormItemArrayProps): JSX
 interface QuestionnaireFormArrayContentProps {
   item: QuestionnaireItem;
   index: number;
-  answers: Record<string, QuestionnaireResponseItemAnswer[]>;
-  responseItems: QuestionnaireResponseItem[];
+  responses: QuestionnaireResponseItem[];
```

Comment:
How should I interpret `responses` vs `currentResponseItems`?  

Is `responses` "all" responses for the entire Questionnaire, and `currentResponseItems` just the items for this particular item?

If we need both, maybe we can brainstorm on naming that clarifies.

Reply:
Or is this what you're referring to in your comment above?  If so, then disregard